providers/devTool.csv

```diff
@@ -1,35 +1,177 @@
+codeslaw,Codeslaw,Allow developers to search for and explore verified smart contract code across the Ethereum ecosystem,"[""Smart Contract Search Engine"", ""Verified Contract Code"", ""Web3 Development""]",$0 ,"Developer tool and search engine that allows developers to search for and explore verified smart contract code across various blockchain networks, primarily Ethereum",FALSE,"[""[Website](https://www.codeslaw.app/)""]"
+wake,Wake,Development and testing framework specifically designed for Solidity,"[""Testing framework based on pytest"", ""Vulnerability & code quality detectors"", ""Static analysis framework""]",Hybrid pricing model,"Provides a suite of developer tools focused on the development, testing, and security analysis of Solidity smart contracts, serving primarily as a development and testing framework using Python.",FALSE,"[""[Website](https://getwake.io/)""]"
+replit,Replit,AI-powered cloud development platform ,"[""AI coding assistant"", ""Programming playground"", ""Cloud IDE"", ""Online code editor"", ""Collaborative coding""]",Hybrid pricing model,"AI-powered, cloud-based platform that provides an integrated development environment (IDE) for building, testing, deploying, and collaborating on software applications directly in the browser.",FALSE,"[""[Website](https://replit.com/)""]"
+toolblox,Toolblox,No-code smart contract and decentralized application (DApp) builder. ,"[""Software development kit (SDK)"", ""API development"", ""Programming utility"", ""Coding tools""]",Hybrid pricing model,"No-code platform for building, deploying, and managing smart contracts and decentralized applications (DApps) on various blockchain networks.",FALSE,"[""[Website](https://www.toolblox.net/)""]"
+coinsdo,CoinsDo,Web3 wallet and digital asset management infrastructure. ,"[""Wallet as a Service (WaaS)"", ""Digital asset wallet solutions"", ""Cryptocurrency wallet API""]",Custom,"Provides a suite of developer tools, primarily a Wallet-as-a-Service (WaaS) solution, that allows developers and businesses to integrate secure, non-custodial digital asset management and transaction capabilities into their own applications.",FALSE,"[""[Website](https://www.coinsdo.com/en)""]"
+mpcvault,MPCVault,Digital asset custody and wallet infrastructure,"[""transaction simulation"", ""API integration"", ""expense management"", ""hardware security""]",Hybrid pricing model,Digital asset custody and wallet infrastructure,FALSE,"[""[Website](https://mpcvault.com/)""]"
+ape,Ape (ApeWorX),Framework,"[""Python"",""Framework""]",$0 ,Smart contract development toolchain for Python developers streamlining testing and deployment,FALSE,"[""[Docs](https://docs.apeworx.io/ape/stable/)""]"
+arbitrum-sdk,Arbitrum SDK,SDK,"[""TypeScript"",""Bridge SDK""]",$0 ,TypeScript SDK for bridging assets and sending cross‑chain messages on Arbitrum chains,FALSE,"[""[Docs](https://docs.arbitrum.io/sdk/)""]"
```

Comment:
If Arbitrum-SDK is used only in Arbitrum and you transfer it to the network level - then it have to be removed from provider level.